RussianFilter/sections/specific.txt

```diff
@@ -49,6 +49,8 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+||ero-top.info/uploads/banner*.jpg
+ero-top.info##.remove_in_mobile
```

Comment:
I meant to exchange this `ero-top.info##.remove_in_mobile` rule for blocking the request :)

Reply:
Oh, I mistook. Removed.